recipes/gdal/all/conanfile.py

```diff
@@ -385,6 +422,10 @@ def _edit_nmake_opt(self):
             self._replace_in_nmake_opt("#CHARLS_LIB=e:\\work\\GIS\gdal\\supportlibs\\charls\\bin\\Release\\x86\\CharLS.lib", "CHARLS_LIB=")
         # Inject required systems libs of dependencies
         self._replace_in_nmake_opt("ADD_LIBS	=", "ADD_LIBS={}".format(" ".join([lib + ".lib" for lib in self.deps_cpp_info.system_libs])))
+        # Trick to enable OpenCL (option missing in upstream nmake files)
+        if self.options.with_opencl:
+            tools.replace_in_file(os.path.join(self._source_subfolder, "alg", "makefile.vc"),
+                                  "$(GEOS_CFLAGS)", "$(GEOS_CFLAGS) /DHAVE_OPENCL")
```

Comment:
If there's no option upstream, why does conan need one?

Reply:
GDAL doesn't have a unified build system, 2 build system (autotools and nmake) are maintained. This option exists in configure.ac, but not in nmake files because these files are not always updated.

Reply:
Thank you for the explanation!